lib/ui/widgets/user_settings.dart

```diff
@@ -35,18 +41,9 @@ class UserSettings extends StatelessWidget {
               keyboardType: TextInputType.text,
               controller: displayNameController,
             ),
-            TextField(
-              decoration: InputDecoration(labelText: 'Photo Url'),
-              keyboardType: TextInputType.url,
-              controller: photoUrlController,
-            ),
-            SizedBox(
-              height: 20,
-            ),
-            Container(
-              width: 120,
-              height: 120,
-              child: notifier.busy ? _imageLoading : _userImage(photoUrlController.text),
+            UserImagePreviewer(
+              photoUrlController: photoUrlController,
```

Comment:
photoUrlController -> can just pass the text over ?

Reply:
can not

Reply:
pls explain why